vars/runIntegTestScript.groovy

```diff
@@ -38,13 +38,14 @@ void call(Map args = [:]) {
     }
     echo "Switch User to Non-Root (uid=1000): ${switchUser}"
 
-    String switchCommandStart = switchUser.equals('true') ? 'su `id -un 1000` -c "' : ''
+    // Avoid issue related to docker ENV keyword is not correctly interpreted in su commands
+    // https://github.com/opensearch-project/opensearch-build-libraries/issues/197
+    String switchCommandStart = switchUser.equals('true') ? "su `id -un 1000` -c \"env PATH=\$PATH $javaHomeCommand" : "env PATH=\$PATH $javaHomeCommand"
```

Comment:
Still not getting how PATH=$PATH matters if javaHomeCommand is added.
`env PATH=\$PATH JAVA_HOME=/opt/java/${javaVersion}` Should be 
`env PATH=\$PATH:/opt/java/${javaVersion}`

Reply:
No need, gradle is only looking for JAVA_HOME or java command in path.

Reply:
Even for the case of adding in path, it still needs to be running as ` env PATH=\$PATH:/opt/java/${javaVersion}/bin` not ` env PATH=\$PATH:/opt/java/${javaVersion}`

Reply:
I see! So we are adding JAVA_HOME as env and not adding java home path.

Reply:
yes.

Reply:
Can you add test case where java home is added? All above tests only add PATH?